libfreerdp-utils/time.c

```diff
@@ -0,0 +1,69 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol client.
+ * Time Utils
+ *
+ * Copyright 2018 Idan Freiberg <speidy@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <time.h>
+
+#include <freerdp/utils/time.h>
+
+
+/**
+ * Set client time zone information.
+ * @param settings settings
+ */
+
+void time_set_client_time_zone(rdpSettings* settings)
```

Comment:
If I'm not mistaken, this would use the server's timezone, and not the clients?

Reply:
right, but its only used when running `xfreerdp` command line client.

before this PR , the protocol parsing was also calling this function at parsing time (https://github.com/Osirium/NeutrinoRDP/pull/4/commits/cd0fa004843c07dba754f273917ca977ec21038b#diff-ba57ae8669b40c994bc78d9639da7b9cL178)

Iv'e removed it since it was doing the wrong thing of passing the server time instead of what passed by the client

Reply:
what happens now is that `xrdp-neutrinordp` module is passing the timezone settings and NeutrinoRDP is just passing them through